spec/fixtures/test.asar/repack.js

```diff
@@ -0,0 +1,22 @@
+// Use this script to regenerate these fixture files
```

Comment:
Perhaps instead of commiting `.asar` files to the repo, tests can create the asar files on the fly?

Reply:
In principal, yes, in reality symbolic links on windows via git are questionable and operate differently depending on OS and filesystem settings

Reply:
hm, i'm confused why symlinks would be relevant to this?

Reply:
ASAR files support symlinks, many of these fixture files have symlinks in them

Reply:
ah, i see.

🤔   can we synthesize the input to `asar` as a data structure in memory rather than on disk? I don't think the asar module as-is supports that... probably overkill to work around this 😞

Reply:
Technically we could probably add the capability to the ASAR module itself.  Like we could expose the `FileSystem` structure in the `asar` module and manually add bytes as files / links to the FS struct.  I don't want to dive down that rabbit hole in this PR though